scripts/pre_commit_linter.py

```diff
@@ -1273,16 +1388,16 @@ def _check_sorted_dependencies(self):
                                 'following manner: dollar imports, regular '
                                 'imports and constant imports, all in sorted '
                                 'order.'
-                                % (property_value, filepath))
+                                % (property_value, filename))
                         if sorted_imports != literal_args:
                             failed = True
                             print (
-                                'Please ensure that in %s in file %s, the '
-                                'stringfied dependencies should be in the '
-                                'following manner: dollar imports, regular '
-                                'imports and constant imports, all in sorted '
-                                'order.'
-                                % (property_value, filepath))
+           	                    'Please ensure that in %s in file %s, the '
```

Comment:
This and the following lines should be indented by 4 from the previous line.

Reply:
This is very strange. I did not make this change as you can see in the screenshot:
![aa](https://user-images.githubusercontent.com/32506039/54496106-75bb6d80-4911-11e9-81be-91d0ff4be3cf.png)

Reply:
It probably happened due to the merge, or because your local branch is behind develop. Make sure to always check the "Files Changed" tab after you push; that's what counts.

Reply:
Oh, actually, I responded thinking that this was a different comment, sorry. I agree, that's very strange. I don't know what happened there.

The advice about checking the "Files Changed" tab still holds though, it would catch things like this.

Reply:
Actually, it's still wrong. Maybe check the "files changed" tab and tweaking until it's right?

Make sure your local env is using only spaces and no tabs.